test/integration/default/controls/service.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
+service_name = 'strongswan'
+service_name = 'strongswan-starter' if (os[:name] == 'ubuntu') && os[:release].start_with?('20')
```

Comment:
As suggested by `rubocop -a --safe`:

```bash
strongswan-formula$  rubocop -a --safe
...
Offenses:

test/integration/default/controls/service.rb:4:37: C: [Corrected] Style/IfUnlessModifier: Modifier form of if makes the line too long.
service_name = 'strongswan-starter' if (os[:name] == 'ubuntu') && os[:release].start_with?('20')
                                    ^^

7 files inspected, 1 offense detected, 1 offense corrected
```

So that becomes:

```suggestion
if (os[:name] == 'ubuntu') && os[:release].start_with?('20')
  service_name = 'strongswan-starter'
end
```